Mu2eG4/src/Mu2eGammaDaughterProcess.cc

```diff
@@ -0,0 +1,99 @@
+// Special process to kill events with low energy photon daughters
+//
+// Original author M. MacKenzie
+
+// Mu2e includes
+#include "Mu2eG4/inc/Mu2eGammaDaughterProcess.hh"
+#include "Mu2eG4/inc/Mu2eG4UserHelpers.hh"
+
+// G4 includes
+#include "G4ios.hh"
+#include "G4VParticleChange.hh"
+#include "G4Track.hh"
+#include "G4Step.hh"
+
+namespace mu2e{
+
+  Mu2eGammaDaughterProcess::Mu2eGammaDaughterProcess(const G4String& aName)
+    : G4VDiscreteProcess(aName,fUserDefined)
```

Comment:
This constructor leaves minDaughterEnergy_ uninitialized.

Reply:
Best practice is to initialize variables in the initialization part of the constructor, not the body, if possible.

Reply:
Good catch! I've updated this with the constructor update.